### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

**Comment by lewisgoddard** [comment]
test this please

**Comment by lewisgoddard** [comment]
Are you all done with this branch, or are you still working through it?

**Comment by waldyrious** [comment]
let me just finish reading the doc, I might do some more tweaks :) just a couple minutes.

**Comment by waldyrious** [comment]
All done now @lewisgoddard :)

File: docs/human-interface-guidelines.md [hunk 1]
```diff
@@ -788,11 +788,11 @@ However there are exceptions. Many icons (especially mimetype icons) have ascend
 <img title="First composition exception example" class="hig-icons-example" style="background-image:url(/images/docs/human-interface-guidelines/icons/exception-icon1.png);" src="/images/docs/human-interface-guidelines/icons/grid.svg">
 <img title="Second composition exception example" class="hig-icons-example" style="background-image:url(/images/docs/human-interface-guidelines/icons/exception-icon2.png);" src="/images/docs/human-interface-guidelines/icons/grid.svg">
 
-Rounder components will always need to be slightly larger than their rectangular counterparts due to the necessary addition of overshoot to compensate for the optical illusion that makes them look smaller.
+Rounder components will generally require some overshoot, to compensate for the optical illusion that makes them look smaller than their rectangular counterparts.
 
 ## Outlines &amp; Contrast {#contrast}
 
-All elementary icons have a thin outline (stroke) to improve their contrast. This stroke width is **one** pixel for each and every icon, at every size.
+All elementary icons have a thin outline (stroke) to improve their contrast. This width of this stroke is **one** pixel for all icons, at every size.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: > This width of this stroke is **one** pixel for all icons...

Comment (waldyrious) [inline docs/human-interface-guidelines.md]: Good catch! Will change.

**Comment by lewisgoddard** [comment]
test this please

**Pull request merged**